examples/pom.xml

```diff
@@ -10,7 +10,8 @@
   <name>examples</name>
   <url>http://maven.apache.org</url>
   <properties>
-    <grpc.version>1.7.0-SNAPSHOT</grpc.version><!-- CURRENT_GRPC_VERSION -->
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <grpc.version>1.6.1</grpc.version><!-- CURRENT_GRPC_VERSION -->
```

Comment:
It looks like this part is stale now since we've bumped the gRPC version

Reply:
oops. didn't mean to check that in.